src/lib/structures/Command.ts

```diff
@@ -88,14 +88,14 @@ export class Command<PreParseReturn = Args, O extends Command.Options = Command.
 		this.fullCategory = options.fullCategory ?? this.location.directories;
 		this.typing = options.typing ?? true;
 
-		this.lexer.setQuotes(
-			options.quotes ?? [
+		this.lexer = new Lexure.Lexer({
```

Comment:
```
this.lexer.quotes = ...
```

It's a public property, which is why there is no setter.

Reply:
It's also because you can pass them in options.

Reply:
I can't do that, [`quotes` is readonly.](https://github.com/sapphiredev/utilities/blob/c4128129c81bd7d02ca98fa61914ed4a59ed98bb/packages/lexure/src/lib/lexer/Lexer.ts#L5)

Reply:
will be fixed with https://github.com/sapphiredev/utilities/pull/403

Reply:
nevermind this, I missed that this is in the constructor